### PR TITLE
Added xuply.dev presence

### PR DESCRIPTION
**Pull request by baybeku**
Xuply is a simple code sharing platform for discord bots and more.

### CODE REVIEW

File: websites/X/Xuply/presence.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    presenceData.details = "Bir profil görüntülüyor:";
+    presenceData.state =
+      document.querySelector("body > main > div > div > div > div > div > h2").textContent +
+      " " +
+      "adlı kullanıcının profilini görüntülüyor!";
+    presenceData.startTimestamp = browsingStamp;
+  } else if(page.startsWith('/code')) {
+	presenceData.details = "Bir kod görüntülüyor:";
+	presenceData.state = document.querySelector("body > main > div > div > div > div > div > h2").textContent + " " + "adlı kodu görüntülüyor."
+    presenceData.startTimestamp = browsingStamp;
```
Comment (ririxichan) [inline websites/X/Xuply/presence.ts:45]: fix tabs.

**Review by ririxichan — CHANGES_REQUESTED** [review]

File: websites/X/Xuply/presence.ts (hunk 1 above)
Comment (baybeku) [reply websites/X/Xuply/presence.ts]: Fixed

File: websites/X/Xuply/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "url": "xuply.dev",
+  "description": {
+    "tr": "Xuply discord botları ve daha fazlası için basit bir kod paylaşım platformudur.",
+    "en": "Xuply is a simple code sharing platform for discord bots and more."
+  },
+  "service": "Xuply",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/fMUpAe0.png",
+  "thumbnail": "http://i.imgur.com/a/AiGDoAD.png",
+  "color": "#0bc9e7",
```
Comment (Bas950) [inline websites/X/Xuply/dist/metadata.json:16]: wrong url it should be: https://i.imgur.com/wTuunx3.png

File: websites/X/Xuply/presence.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
+const presence = new Presence({
+  clientId: "760926411090690074",
+});
```
Comment (Bas950) [inline websites/X/Xuply/presence.ts:3]: no trailing comma

File: websites/X/Xuply/presence.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "xuply",
```
Comment (Bas950) [inline websites/X/Xuply/presence.ts:7]: no trailing comma

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/X/Xuply/presence.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
+const presence = new Presence({
+  clientId: "760926411090690074"
+});
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "xuply",
```
Comment (ririxichan) [inline websites/X/Xuply/presence.ts]: it still is not fixed.

**Review by ririxichan — CHANGES_REQUESTED** [review]

**Review by ririxichan — APPROVED** [review]

File: websites/X/Xuply/presence.ts (hunk 5 above)
Comment (baybeku) [reply websites/X/Xuply/presence.ts]: now fixed

File: websites/X/Xuply/presence.ts (hunk 3 above)
Comment (baybeku) [reply websites/X/Xuply/presence.ts]: Fixed

File: websites/X/Xuply/dist/metadata.json (hunk 2 above)
Comment (baybeku) [reply websites/X/Xuply/dist/metadata.json]: Fixed

File: websites/X/Xuply/presence.ts (hunk 4 above)
Comment (baybeku) [reply websites/X/Xuply/presence.ts]: Fixed

**Comment by Bas950** [comment]
Website isn't older than 2 months.

**Pull request closed without merging**